{"name": "Проверка нельзя создать пользователя, который уже зарегистрирован", "status": "passed", "description": "Повторный запрос на создание пользователя, проверяем ответ", "start": 1730171428139, "stop": 1730171428433, "uuid": "43ed755a-cd23-429f-b20b-fb3fd20cd623", "historyId": "252e57feb76398321a64ee63034cb412", "testCaseId": "252e57feb76398321a64ee63034cb412", "fullName": "tests.test_create_user.TestCreateUser#test_registration_double_user_failed", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_user"}, {"name": "subSuite", "value": "TestCreateUser"}, {"name": "host", "value": "MacBook-Pro-user.local"}, {"name": "thread", "value": "1205-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_user"}]}